workspaces/vscode-cpp/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
A note in the README that this is the VSCode settings would be great.

Reply:
That said: I believe the vast majority of instructors don't have a reason to change the settings. I don't think it's worth adding that in the notes. The existing README notes are right now focused on what is important for most users.